frontend/packages/core/src/stepper.tsx

```diff
@@ -183,19 +195,30 @@ const StepIcon: React.FC<StepIconProps> = ({ index, variant }) => {
 export interface StepProps {
   label: string;
   error?: boolean;
+  isComplete?: boolean;
 }
 /* eslint-enable react/no-unused-prop-types */
 
 const Step: React.FC<StepProps> = ({ children }) => <>{children}</>;
 
-export interface StepperProps extends Pick<MuiStepperProps, "orientation"> {
+export interface StepperProps
+  extends Pick<MuiStepperProps, "orientation">,
+    Pick<MuiStepperProps, "nonLinear"> {
```

Comment:
You can combine these, ie:

```suggestion
  extends Pick<MuiStepperProps, "orientation" | "nonLinear"> {
```